app/routes.py

```diff
@@ -1,2 +1,218 @@
+from app import db
 from flask import Blueprint
+from flask import request, Blueprint, make_response
+from flask import jsonify
+from .models.goal import Goal
+from .models.task import Task
+from datetime import datetime
+import os
+import requests
+from dotenv import load_dotenv
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+@tasks_bp.route("", methods=["POST", "GET"])
+def tasks_functions():
+    if request.method == "POST":
+        req_body = request.get_json()
+        if "title" not in req_body or "description" not in req_body or "completed_at" not in req_body:
+            return {
+                "details": "Invalid data"
+            }, 400
+        new_task = Task(title = req_body["title"], description = req_body["description"], completed_at = req_body["completed_at"])
+        db.session.add(new_task)
+        db.session.commit()
+        response_body = {
+            "task": new_task.to_json()
+        }
+        return jsonify(response_body), 201
+
+    elif request.method == "GET":
+        order = request.args.get("sort")
+        if order == "asc":
+            new_order = Task.query.order_by(Task.title.asc())
+            response = []
+            for task in new_order:
+                response.append(task.to_json())
+            return jsonify(response), 200
+
+        elif order == "desc":
+            new_order = Task.query.order_by(Task.title.desc())
+            response = []
+            for task in new_order:
+                response.append(task.to_json())
+            return jsonify(response), 200
+
+        all_tasks = Task.query.all()
+        response_body = []
+        for any_task in all_tasks:
+            response_body.append(any_task.to_json())
+        return jsonify(response_body), 200
```

Comment:
Optional: you could add this code in an else statement

```suggestion
    else:
        all_tasks = Task.query.all()
        response_body = []
        for any_task in all_tasks:
            response_body.append(any_task.to_json())
        return jsonify(response_body), 200
```